Nimble_Snapshots/DynamicSize/DynamicSizeSnapshot.swift

```diff
@@ -152,7 +152,8 @@ public func haveValidDynamicSizeSnapshot(named name: String? = nil,
                                          usesDrawRect: Bool = false,
                                          pixelTolerance: CGFloat? = nil,
                                          tolerance: CGFloat? = nil,
-                                         resizeMode: ResizeMode = .frame) -> Predicate<Snapshotable> {
+                                         resizeMode: ResizeMode = .frame,
+                                         isIgnoreScale: Bool = false) -> Predicate<Snapshotable> {
```

Comment:
At least for me, the parameter name does not make sense. A better name could be `shouldIgnoreScale`, `isIgnoringScale` or `ignoreScale` (the first one being the one I like more).

Reply:
Yeah that's a fair point. `shouldIgnoreScale` makes sense to me.